protos/sources/vega/markets.proto

```diff
@@ -295,6 +295,10 @@ message Market {
   string linear_slippage_factor = 13;
   // Quadratic slippage factor is used to cap the slippage component of maintenance margin - it is applied to the square of the slippage volume.
   string quadratic_slippage_factor = 14;
+  // ID of the market this market succeeds
```

Comment:
```suggestion
  // ID of the market that the successor should take over from.
```

Reply:
This is as part of a market type, not the proposal anymore. This is now a proposal that is set up and ready to succeed a market, and is a market in its own right

Reply:
Oh - I see!